apps/hash-frontend/src/pages/[shortname].page/pinned-entity-type-tab-contents.tsx

```diff
@@ -0,0 +1,358 @@
+  selectClasses,
+  Skeleton,
+  styled,
+  Typography,
+} from "@mui/material";
+import {
+  format,
+  formatDistanceToNowStrict,
+  isBefore,
+  subWeeks,
+} from "date-fns";
+import {
+  Fragment,
+  FunctionComponent,
+  Ref,
+  useCallback,
+  useMemo,
+  useState,
+} from "react";
+
+import { useAccountPages } from "../../components/hooks/use-account-pages";
+import { useCreatePage } from "../../components/hooks/use-create-page";
+import { generateEntityLabel } from "../../lib/entities";
+import { Org, User } from "../../lib/user-and-org";
+import { ArrowDownAZRegularIcon } from "../../shared/icons/arrow-down-a-z-regular-icon";
+import { ArrowUpZARegularIcon } from "../../shared/icons/arrow-up-a-z-regular-icon";
+import { ClockRegularIcon } from "../../shared/icons/clock-regular-icon";
+import { PageLightIcon } from "../../shared/icons/page-light-icon";
+import { PlusRegularIcon } from "../../shared/icons/plus-regular";
+import { Button, Link, MenuItem } from "../../shared/ui";
+import { useEntityIcon } from "../../shared/use-entity-icon";
+import { ProfilePageTab } from "./util";
+
+const EntityRow: FunctionComponent<{
+  entity: Entity;
+  profile: User | Org;
+  entitiesSubgraph: Subgraph<EntityRootType>;
+}> = ({ entity, profile, entitiesSubgraph }) => {
+  const label = generateEntityLabel(entitiesSubgraph, entity);
+
+  const href = `/@${profile.shortname}/${
+    entity.metadata.entityTypeId === types.entityType.page.entityTypeId
+      ? ""
+      : "entities/"
+  }${extractEntityUuidFromEntityId(entity.metadata.recordId.entityId)}`;
+
+  const updatedAt = new Date(
+    entity.metadata.temporalVersioning.decisionTime.start.limit,
+  );
+
+  const updatedAtHumanReadable = isBefore(updatedAt, subWeeks(new Date(), 1))
+    ? format(updatedAt, "d MMMM yyyy")
+    : `${formatDistanceToNowStrict(updatedAt)} ago`;
+
+  const icon = useEntityIcon({
+    entity,
+    pageIcon: <PageLightIcon sx={{ fontSize: 18 }} />,
+  });
+
+  return (
+    <Link
+      target="_blank"
+      noLinkStyle
+      href={href}
+      sx={{
+        h2: {
+          transition: ({ transitions }) => transitions.create("color"),
+        },
+        "&:hover": {
+          h2: {
+            color: ({ palette }) => palette.blue[70],
+          },
+        },
+      }}
+    >
+      <Box sx={{ padding: 3 }}>
+        <Box
+          display="flex"
+          alignItems="center"
+          columnGap={1.5}
+          sx={{
+            "> svg": {
+              color: ({ palette }) => palette.gray[50],
+            },
+          }}
+        >
+          {icon}
+          <Typography component="h2" sx={{ fontWeight: 700, fontSize: 14 }}>
+            {label}
+          </Typography>
+          <Typography
+            sx={{
+              color: ({ palette }) => palette.gray[90],
+              fontSize: 11,
+              fontWeight: 500,
+              textTransform: "uppercase",
+            }}
+          >
+            {updatedAtHumanReadable}
```

Comment:
Could we have a tooltip here with the full date?

Reply:
Yes, added in https://github.com/hashintel/hash/pull/3338/commits/470d40000fa0417c0a615e4511664fe2cce7830a

Looks like this:
<img width="530" alt="image" src="https://github.com/hashintel/hash/assets/42802102/e867d7aa-40f5-4ee5-94e3-14343cffc443">

Reply:
Looks good, thank you